core/services/prom_reporter_test.go

```diff
@@ -39,6 +44,33 @@ func Test_PromReporter_OnNewLongestChain(t *testing.T) {
 
 		backend.On("SetUnconfirmedTransactions", int64(3)).Return()
 		backend.On("SetMaxUnconfirmedBlocks", int64(35)).Return()
+		backend.On("SetPipelineTaskRunsQueued", 0).Return()
+		backend.On("SetPipelineRunsQueued", 0).Return()
+
+		head := models.Head{Number: 42}
+		reporter.OnNewLongestChain(context.Background(), head)
+
+		backend.AssertExpectations(t)
+	})
+
+	t.Run("with unfinished pipeline task runs", func(t *testing.T) {
+		store, cleanup := cltest.NewStore(t)
+		defer cleanup()
+
+		require.NoError(t, store.DB.Exec(`SET CONSTRAINTS pipeline_task_runs_pipeline_run_id_fkey DEFERRED`).Error)
```

Comment:
we don't need this anymore with the migration right?

Reply:
Yes we do. This is required in addition to the migration